qiling/utils.py

```diff
@@ -485,7 +485,7 @@ def os_setup(archtype, ostype, ql):
 
 
 def profile_setup(ostype, profile, ql):
-    logging.debug("Customized profile: %s" % profile)
+    debugmsg = "Customized profile: %s" % profile
```

Comment:
Because this was logged before logging initialization, there was a default console logger appended.
When the logging is initialized first, this does not happen and only the registered loggers will be present.